include/sampleflow/consumers/spurious_autocovariance_dim_n.h

```diff
@@ -0,0 +1,289 @@
+// ---------------------------------------------------------------------
```

Comment:
Since the name of the class is simply `SpuriousAutocovariance`, the appropriate file name should be `spurious_autocovariance.h`.

Reply:
fixed